AsyncDisplayKit/ASViewController.m

```diff
@@ -113,7 +149,9 @@ - (void)setAutomaticallyAdjustRangeModeBasedOnViewEvents:(BOOL)automaticallyAdju
 - (void)updateCurrentRangeModeWithModeIfPossible:(ASLayoutRangeMode)rangeMode
```

Comment:
In the case of not adding the ifNeeded/ifPossible, would you do `updateCurrentRangeModeWithRangeMode`?